scripts/test_lua.rb

```diff
@@ -1,10 +1,8 @@
-#!/usr/bin/ruby
+#!/usr/bin/env ruby
 
-LUA_DIR     = "/Users/jkoppel/research_large/other_frameworks/lua/"
-PATH_TO_LUA = LUA_DIR + "src/lua"
-LUA_TESTS   = LUA_DIR + "lua-5.3.3-tests/"
-
-RUNPROG = ".stack-work/dist/x86_64-osx/Cabal-3.8.1.0/build/examples-multi/examples-multi"
+# In order to override env values put them in the devenv.local.nix
+# file in the projet root diretory.
+RUNPROG = "cabal run cubix-examples:multi --"
```

Comment:
This might not make a difference for this one, because the tests only run once, as a whole. But in general this is far slower compared to calling the executable directly.